activejob/lib/active_job/logging.rb

```diff
@@ -39,7 +39,7 @@ module Logging #:nodoc:
 
     private
       def tag_logger(*tags)
-        if logger.respond_to?(:tagged)
+        if ActiveJob::Base.logger.respond_to?(:tagged)
```

Comment:
Would not be better to `tag_logger` use the `logger` of the instance instead of `ActiveJob::Base`?